bin/mark-non-labs-packages-private.js

```diff
@@ -23,7 +23,7 @@ async function markNonLabsPackagesPrivate() {
     const dir = path.relative(pkg.rootPath, pkg.location);
     if (dir.startsWith('labs')) continue;
     const data = readJsonFile(pkg.manifestLocation);
-    let modified = !data.private;
+    const modified = !data.private;
```

Comment:
Can we update the base branch with this change first?

Reply:
good catch! updated in https://github.com/strongloop/loopback-next/pull/3150